packages/template-retail-react-app/app/components/_app-config/index.jsx

```diff
@@ -55,6 +60,7 @@ const AppConfig = ({children, locals = {}}) => {
             redirectURI={`${appOrigin}/callback`}
             proxy={`${appOrigin}${commerceApiConfig.proxyPath}`}
             headers={headers}
+            // Uncomment 'clientSecret' to use SLAS private client login flows
             // clientSecret="enable-client_secret"
```

Comment:
Should we have `_PLACEHOLDER_PROXY-PWA_KIT_SLAS_CLIENT_SECRET` here?

Reply:
On a second thought, what I assume we have this line commented out is for developers going from public to private slas? I think the documentation will have a migration section and this line will be mentioned here. I don't think this line is necessary.

Reply:
This placeholder inside template-retail-react-app is more for us since we are the ones who will start the template directly.

Project developers who create their projects using pwa-kit-create-app will get a different version derived from the copy of the _app-config template in the pwa-kit-create-app package.